curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -46,6 +46,21 @@ static FuseClientOption *fuseClientOption = nullptr;
 
 DECLARE_int32(v);
 
+void EnableSplice(struct fuse_conn_info* conn) {
```

Comment:
Splicing has requirements for the kernel version. You need add enable/disable flag and remarks

Reply:
`capable` in fuse_conn_info indicates capability flags that the kernel supports, and below if conditions are used to do the right thing, like, if kernel support `FUSE_CAP_SPLICE_MOVE` then I want this feature.